views/result.erb

```diff
@@ -0,0 +1,13 @@
+<link rel="stylesheet" type="text/css" href="/style.css"/>
+<div id="allcontent">
+  <h1>Rock Paper Scissors<h1>
```

Comment:
Great initiative using a separate page to display the results and give the user the opportunity to play the game again